### PR TITLE
[Bromley] Add garden waste container request.

### PR DESCRIPTION
**Pull request by dracos**
[skip changelog]
Fixes https://github.com/mysociety/fixmystreet-commercial/issues/2228

### CODE REVIEW

**Review by struan — APPROVED** [review]
👍

**Comment by dracos** [comment]
@struan Actually, after sleeping on it, this isn't ideal because if two reports at Echo end, they're both going to be pushing updates, so we really need two reports at this end don't we. So will change this to create two reports (like it was two different types of container, but both same type with different stuff to do).

**Comment by dracos** [comment]
@struan Okay, this should now create two reports for the damaged case (one Deliver, one Remove); Open311 end I'll change to accept first of submitted data rather than last, as discussed. Hopefully ready to review now.

File: perllib/FixMyStreet/App/Controller/Waste.pm [hunk 1]
```diff
@@ -213,6 +218,14 @@ sub process_request_data : Private {
         $data->{detail} = "Quantity: $quantity\n\n$address";
         $c->set_param('Container_Type', $id);
         $c->set_param('Quantity', $quantity);
+        if ($reason eq 'damaged') {
+            $c->set_param('Action', 2); # Remove XXX
+            $c->set_param('Reason', 3); # Damaged XXX
+            $c->forward('add_report', [ $data ]) or return; # Additional remove report
+            $c->set_param('Action', 1); # Deliver
+        } elsif ($reason eq 'stolen' || $reason eq 'taken') {
+            $c->set_param('Reason', 1); # Missing / Stolen XXX
```
Comment (struan) [inline perllib/FixMyStreet/App/Controller/Waste.pm:227]: might be nice to have a test for this bit too.

**Review by struan — APPROVED** [review]
🗑️

**Pull request merged**